src/dashboard/get_from_parquet/get_from_parquet.py

```diff
@@ -30,7 +30,16 @@ def from_parquet_handler(event, context):
         # TODO: this should be converted to a streamingresponse at some point
         # https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/fastapi-response-streaming
         return functions.http_response(
-            200, payload, extra_headers={"Content-Type": "text/csv"}, skip_convert=True
+            200,
+            payload,
+            extra_headers={
+                "Content-Type": "text/csv",
+                "Content-disposition": (
+                    f"attachment; filename={s3_path.split('/')[-1].replace('.parquet','.csv')}"
+                ),
+                "Content-Length": len(payload.encode("UTF-8")),
```

Comment:
This feels odd - I would expect `len(payload)` (i.e. the length of the bytes sent, not the resulting utf8 string)